dataeng/jobs/createJobs.groovy

```diff
@@ -21,8 +22,7 @@ try {
 }
 
 def taskList = [
-    AnalyticsEmailOptinJob,
-    AnalyticsExporterJob
+    UserActivityJob
```

Comment:
I'm assuming that AnalyticsEmailOptinJob and AnalyticsExporterJob shouldn't really be removed.